docs/source/installation.md

```
@@ -49,6 +49,25 @@ To install the [current milestone release](https://pypi.org/project/monai/):
 pip install monai
 ```
 
+MONAI supports the extras syntax such as `pip install 'monai[nibabel]'`. The options are
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)